EventFilter/DTRawToDigi/plugins/DTDDUUnpacker.h

```diff
@@ -11,7 +11,7 @@
 
 #include <EventFilter/DTRawToDigi/plugins/DTUnpacker.h>
 #include <EventFilter/DTRawToDigi/plugins/DTROS25Unpacker.h>
-
+#include "DataFormats/DTDigi/interface/DTControlData.h"
```

Comment:
@mrodozov : since you are there, there are a few includes that can be removed from EventFilter/DTRawToDigi/plugins/DTDDUUnpacker.cc, then...